ref/ExcludeAttributeList.txt

```diff
@@ -0,0 +1 @@
+T:System.Runtime.CompilerServices.CompilerGeneratedAttribute
```

Comment:
Can you add `System.Diagnostics.DebuggerStepThroughAttribute` while we're at it? That one is also not interesting to API consumers and can change depending on our whims.